README.md

```diff
@@ -0,0 +1,16 @@
+# python-ethtool
+
+*Python bindings for the ethtool kernel interface*
+
+**This is a new upstream for python-ethtool maintained by Red Hat/Fedora
```

Comment:
As this is the fedora-python github organization I would suggest to remove the Red hat/ Fedora Python maintainance team line. Maybe just put Fedora's Python SIG?